posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+
+## Custom file associations
+
+First, we added the possibility to associate different external applications to open specific file extensions. By going to the “Files” option in the preferences window, it is possible to set the file associations by, selecting the File Types and the Associated applications that should be used to open this specific file extensions. 
+
+![File associations](/images/spyder-files-file-associations.png)
+
+Once this configuration is set you can go to the files pane in Spyder and whenever you open a file with the selected extension, it will open automatically with the selected associated application. Additionally, when you right click a file with one of this extensions, you will find an “open with” option that will show the application associated for this extension.
+
+![Open With](/images/spyder-files-open-with.png)
+
+## Single click open on file explorer
+
+Another improvement made was the option to open files and directories with a single click which was not possible before because files and folders could only be opened by double clicking them. In order to enable this option, go to the files tab in the preferences window, check the box “Single click to open files” and apply the changes. With this set, now only a single click is needed to open a file externally or in Spyder, from the files tab. 
+
+![Single click](/images/spyder-files-single-click.png)
+
+Bear in mind that changing this configuration option will also affect the behaviour of the Project Explorer pane.
+
+## Open files externally
+
+Additionally, we added a context menu action called Open externally to open files with the Operating System default program associated with the file type. To use this action, go to the file you want to open in the files tab and right-click it to see the context menu. Then select “Open Externally” and the file will be opened outside Spyder, with the default program associated with its file type, depending on the Operating System.
+
+![Open Externally](/images/spyder-files-open-externally.png)
+
+Moreover, now it is possible to select several files to perform a specific action thanks to the multi-select functionality added. For this, press CTRL or SHIFT and Click on the files that are going to be selected without releasing the CTRL/SHIFT. It is also possible to select the files without doing it one by one, but selecting a file with the CTRL/SHIFT clicked and selecting a not contiguous file to this one which will select all the files between them.
+By doing this, it is possible to execute some of the actions available in the context menu for all the files selected including delete, rename, move, copy and open externally.
+
+
+![Select Files](/images/spyder-files-select-files.png)
+
+## Absolute and relative path handling
+
+Another functionality added was the ability to copy and paste files and their absolute or relative paths. To access these actions, go to the context menu by right-clicking a file in the files tab. 
```

Comment:
```suggestion
Spyder 4 now allows you to easily copy the absolute or relative paths of one or more files in the `Files` pane and pasting them as intelligently-formatted text anywhere else in Spyder. Just right-clicking any file or files in the Files or Project panes and select the desired option.
```

* Combine with the following block for better flow
* Make description more accurate
* Prefer active voice
* Clarify and simplify explanation
* Use standardized terminology and match UI text